.travis.yml

```diff
@@ -158,28 +158,36 @@ after_success:
         echo "Publishing from branch $TRAVIS_BRANCH";
         mvn clean deploy -DskipTests=true -B -U -P release --settings CI/settings.xml;
         echo "Finished mvn clean deploy for $TRAVIS_BRANCH";
-        pushd .;
-        cd modules/openapi-generator-gradle-plugin;
-        ./gradlew -Psigning.keyId="$SIGNING_KEY" -Psigning.password="$SIGNING_PASSPHRASE" -Psigning.secretKeyRingFile="${TRAVIS_BUILD_DIR}/sec.gpg" -PossrhUsername="${SONATYPE_USERNAME}" -PossrhPassword="${SONATYPE_PASSWORD}" publishPluginMavenPublicationToSonatypeRepository closeAndReleaseSonatypeStagingRepository;
+        ./gradlew \
+          :openapi-generator-gradle-plugin:publishPluginMavenPublicationToSonatypeRepository \
+          closeAndReleaseSonatypeStagingRepository \
+          -Psigning.keyId="$SIGNING_KEY" \
+          -Psigning.password="$SIGNING_PASSPHRASE" \
+          -Psigning.secretKeyRingFile="${TRAVIS_BUILD_DIR}/sec.gpg" \
+          -PossrhUsername="${SONATYPE_USERNAME}" \
+          -PossrhPassword="${SONATYPE_PASSWORD}";
```

Comment:
Since the root project has a Gradle wrapper, changing the directory is no longer required. I was able to get rid of the calls to `pushd` / `popd`.